Common/Data/SubscriptionDataConfig.cs

```diff
@@ -101,6 +101,11 @@ public Symbol Symbol
         public decimal PriceScaleFactor;
```

Comment:
I did try. 
`PriceScaleFactor` depends on `DataNormalizationMode`. When `DataNormalizationMode` is `Raw`, `PriceScaleFactor = 1` and we don't get the "real" price scale factor to adjust the price that goes into the indicator.
So I created a `UnconditionalPriceScaleFactor` on SubscriptionDataConfig and updated it along with `PriceScaleFactor` in a way that it would get the real price scale factor even if `DataNormalizationMode` is `Raw`. 
It wasn't working. Thus I printed out `PriceScaleFactor` and `UnconditionalPriceScaleFactor` and compare them with the value in the csv factor file. It was off. In the end, I decided to get the value from the `FactorFile` object and it is working.